DQM/CSCMonitorModule/plugins/CSCDQM_MonitorObjectProvider.h

```diff
@@ -115,7 +115,7 @@ namespace cscdqm {
   class MonitorObjectProvider {
 
     public:
-    
+      virtual ~MonitorObjectProvider() {}
```

Comment:
Looks like this either needs to be declared 
```
virtual ~MonitorObjectProvider() nothrow(false) {}
```
or it might work to do
```
virtual ~MonitorObjectProvider();
```
and then define the destructor near the bottom of this file.